ktor-http/common/src/io/ktor/http/parsing/ParseException.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.http.parsing
+
+public class ParseException(
```

Comment:
This should be either documented or internal. Also, please don't forget to run apiDump for this

Reply:
thanks, done